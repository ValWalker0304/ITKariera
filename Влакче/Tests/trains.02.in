Add 48195 Zimnitsa P 1
Add 48506 Sliven P 5
Add 46527 Blagoevgrad P 2
Add 44521 GOryahovitsa P 10
Add 98820 Zimnitsa P 7
Add 99567 Zimnitsa P 6
Add 82182 Ruse P 10
Add 55063 Burgas P 6
Travel
Add 45162 Burgas P 2
Travel
Travel
Travel
Add 94118 Vratsa P 8
Add 50508 Zimnitsa P 3
Travel
Travel
End