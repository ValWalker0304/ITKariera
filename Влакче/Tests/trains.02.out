55063 Burgas P 6
45162 Burgas P 2
82182 Ruse P 10
99567 Zimnitsa P 6
50508 Zimnitsa P 3
94118 Vratsa P 8
